code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -133,6 +130,14 @@
 			to_chat(user, "<span class='notice'>You remove the modifications on [src]. Now it will fire .38 rounds.</span>")
 	return TRUE
 
+/obj/item/gun/ballistic/revolver/detective/modele1892
+	name = "\improper Modèle 1892 revolver"
+	desc = "A french smokeless powder revolver produced by Manufacture d'armes de Saint-Étienne. Fires a special 8mm cartridge."
```

Comment:
"french" should be capitalized (and replaced with "Space French" anyway, no?)

Reply:
"1892 revolver"